docs/guides/map-use-case-to-provider.md

```diff
@@ -18,32 +18,67 @@ Mapping happens between _a specific version of Profile_ and some _Provider_. Cho
+Running the above command creates a new Comlink file and links the new map in your local `super.json` configuration file. The new map will look something like this:
 
 ```hcl title="<profile-name>.<provider-name>.suma"
 profile = "<profile-name>@<version>"
 provider = "<provider-name>"
-
-"""
-Comment for the map to UseCaseName
-"""
-map UseCaseName {} // UseCaseName will be different based on the actual use case in the profile
+// Complete map specification: https://superface.ai/docs/comlink/reference/map
+
+map UsecaseName { // UseCaseName will be different based on the actual use case in the profile
+  // Available input fields:
+  // `input.field` - required string
+  
+  // -----------------------
+
+  // 1) Change HTTP method and path to make an HTTP call
+  // provider base URL https://swapi.dev/api - base url will be different based on the actual url in provider.json
+  http POST "/endpoint/api/{ input.field }" {
+    // 2) Specify security scheme id from Provider JSON definition
+    security none
+  
+    // 3) Pass input values to the HTTP request
+    request {
+      query {
+        // param_name = input.fieldName
+      }
+      headers {}
+      body {}
+    }
+  
+    // 4) Map successful HTTP response to the result. The content type is optional
+    response 200 "application/json" {
+      return map result {
+        outputField = "", // optional nulllable string
+      }
+    
+    }
+  
+    // 5) Optionally map unsuccessful HTTP response to the use case error
+    response 500 {
+      return map error {
+        title = "", // required string
+        detail = "", // optional string
+      }
+    }
+  }
+}
 ```
 
 ## Map use cases
 
-Every profile defines one or more _use cases_. You need to map the use case interfaces to the concrete requests and results towards a provider. If you used CLI to bootstrap the map, it will have pre-defined empty mappings for every profile use case.
+Every profile defines one or more _use cases_. You need to map the use case interfaces to the concrete requests and results towards a provider. If you used CLI to bootstrap the map, it will have pre-defined mappings for every profile use case.
 
 ### Reading use case inputs {#input-object}
 
 Use cases usually define & expect some _inputs_ from the user. These inputs are [defined in profile in the dedicated field](https://superface.ai/docs/comlink/profile#sec-Use-case).
 
-You can access these inputs via `input` object which is available _inside use case mapping_.
+You can access these inputs via `input` object which is available _inside use case mapping_.  If you used CLI to bootstrap the map, prepared map lists all inputs (eq. `// input.field - required string`) defined by use case. Each input is paired with description containing information about field type, if it is required and can be null.
```

Comment:
```suggestion
You can access these inputs via the `input` object which is available _inside a map_.  If you used the CLI to bootstrap the map, the resulting map lists all inputs (eq. `// input.field - required string`) defined by the use case. Each input is paired with its description, containing information about its type, if it is required, and if it can be null.
```